test/acceptance/features/steps/generic_testapp.py

```diff
@@ -33,3 +33,8 @@ def is_running(context, application_name):
 def check_env_var_value(context, name, value):
     found = polling2.poll(lambda: context.application.get_env_var_value(name) == value, step=5, timeout=400)
     assert found, f'Env var "{name}" should contain value "{value}"'
+
+@step(u'The application env var "{name}" does not have value "{value}"')
```

Comment:
see the comment in the scenario. The step should check if env var does not exist at all. For that you cannot use `application.get_env_var_value` - you need to `/env/ENV_VAR` endpoint and repeat the query until you get response with 404 http status code. With that, we can be sure that the env var does not exists.

Reply:
Done